### DISCUSSION TITLE
Deployment code

### DISCUSSION THREAD

**Discussion by jey07** [mindee/doctr#928] [General]
Is there any different version or method of doc ocr which has production deployment code/env which runs fast ?

**Comment by felixdittrich92** [comment]
Hi @jey07 👋 ,
Do you mean something like a optimized graph inference pipeline (for example Onnxruntime) ? If so we work currently on ONNX support for each model and integrating a pipeline for this 👍 
For the moment the fastest solution would be to use the detection and recognition models which uses a mobilenet backbone and run this on a Gpu

**Reply by jey07** [reply]
Hello @felixdittrich92 ,  may i know how much time does it take to detect and predict together using Onnex or in gpu.

**Reply by felixdittrich92** [reply]
@jey07 :wave: 
This question is really hard to answer without knowledge about your environment and files :sweat_smile: 

- For ONNX as explained ONNX support is currently under development, if it is done it will speed up the latency on CPU. 

- We have some benchmarks in the documentation :) 

- To get the fastest latency i would suggest to use db_mobilenet_v3_large as detection model and crnn_mobilenet_v3_small / large as recognition model you can also take an look at this discussion: #876
